src/containers/PaymentMethodsPage/PaymentMethodsPage.duck.js

```diff
@@ -1 +1,95 @@
-//TODO
+import { fetchCurrentUser } from '../../ducks/user.duck';
+import { storableError } from '../../util/errors';
+import * as log from '../../util/log';
+
+// ================ Action types ================ //
+
+export const SETUP_INTENT_REQUEST = 'app/CheckoutPage/SETUP_INTENT_REQUEST';
```

Comment:
Rename these strings. 'app/PaymentMethodsPage/SETUP_INTENT_REQUEST'
(These are for debugging purposes. We can track which calls are sent to reducers and in which order.)